packages/cli/src/license.ts

```diff
@@ -60,7 +60,7 @@ export class License {
 		const isMainInstance = instanceType === 'main';
 		const server = this.globalConfig.license.serverUrl;
 		const offlineMode = !isMainInstance;
-		const autoRenewOffset = this.globalConfig.license.autoRenewOffset;
+		const autoRenewOffset = 60 * 60 * 72; // 72 hours
```

Comment:
Can we please use the constant?

https://github.com/n8n-io/n8n/blob/53e11b19adf8bae2036229f19ee881bc4bf2ea4b/packages/cli/src/constants.ts#L118